crates/volta-core/src/platform/mod.rs

```diff
@@ -220,6 +220,9 @@ mod test {
         let mut pathbufs: Vec<PathBuf> = Vec::new();
         pathbufs.push(volta_home().unwrap().shim_dir().to_owned());
         pathbufs.push(PathBuf::from("C:\\\\somebin"));
+        #[cfg(feature = "volta-updates")]
```

Comment:
Does `cfg_if!` work here? (I don't know, to be clear, it's an actual question!)

```rust
cfg_if! {
    if #[cfg(feature = "volta-updates")] {
        pathbufs.push(volta_install().unwrap().root().to_owned());
    } else {
        pathbufs.push(volta_install().unwrap().bin_dir());
    }
}
```

*If* that works, I think it makes the intent a lot clearer (for as long as this is in here).

(Same question/suggestion for the similar bits below.)

Reply:
It looks like it does! I'll update them to use that, good catch!

Reply:
I lied, that doesn't actually work. All of those blocks were Windows-only, so _of course_ there weren't any compiler errors on Mac 😅

Reply:
Hahahahaha 😭 Worth a try – thanks for checking!